packages/http-client-csharp/eng/pipeline/publish.yml

```diff
@@ -47,3 +47,56 @@ extends:
               inputs:
                 useGlobalJson: true
                 workingDirectory: $(Build.SourcesDirectory)/packages/http-client-csharp
+
+      - stage: CreateAzureSdkForNetPR
+        displayName: Create PR for azure-sdk-for-net
+        dependsOn: CSharp_Publish
+        condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranchName'], 'main'))
```

Comment:
Ah I was trying to test and it is this condition for not being a pull request that caused the stage to be skipped. Can we remove this temporarily for testing?